scripts/pipelines/gcloud/templates/package/package-pipeline.yml.template

```
@@ -0,0 +1,18 @@
+steps:
+- name: 'gcr.io/cloud-builders/gsutil'
+  args: ['cp', 'gs://package-test-bucket/build_1.tar', '.']
+
+- name: ${_DOCKER_IMAGE}
+  args: ['tar', '-xf', 'build_1.tar', '.']
```

```suggestion
- name: ${_DOCKER_IMAGE}
  entrypoint: bash
  args: 
  - -c
  - |
    # In case of more than 1 tar, we extract the most recent one)
    for i in $(ls -t build_*.tar)
    do
        tar -xf $i
        break
    done
```
(not sure about the indentation)